app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -64,12 +73,68 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   @field:[Inject Named("date_for_user_input")]
   lateinit var dateOfBirthFormat: DateTimeFormatter
 
+  @Inject
+  lateinit var numberValidator: PhoneNumberValidator
+
+  @Inject
+  lateinit var dateOfBirthValidator: UserInputDateValidator
+
+  @Inject
+  lateinit var userClock: UserClock
+
+  @Inject
+  lateinit var patientRepository: PatientRepository
+
+  @Inject
+  lateinit var utcClock: UtcClock
+
+  @Inject
+  lateinit var schedulersProvider: SchedulersProvider
+
   @Inject
   lateinit var activity: TheActivity
 
   @Inject
   lateinit var crashReporter: CrashReporter
 
+  private val screenKey by unsafeLazy {
+    screenRouter.key<EditPatientScreenKey>(this)
+  }
+
+  private val viewRenderer by unsafeLazy {
+    EditPatientViewRenderer(this)
```

Comment:
Why is the `viewRenderer` a lazy delegate? Is there a problem if we directly instantiate it? As far as I can tell, the renderer just holds the reference to the view instance, so it should be safe to just instantiate it here.

Reply:
Done.